examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -64,9 +69,31 @@
 goes_ts = ts.TimeSeries(goes_data, meta, units, source="xrs")
 
 ###############################################################################
-# Finally, we can plot the timeseries.
+# NOAA also provides the past 7 days of flare event identifications which we 
+# can also parse.
+# Note that it is possible that no flares occurred in the last 7 days.
+
+flare_events = pd.read_json(
+    "https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json"
+)
+
+###############################################################################
+# Next we sort the list and retain the five largest flares. 
+
+if len(flare_events) > 0:
+    largest_flares = flare_events.sort_values('max_xrlong', ascending=False)[:5]
+else:
+    largest_flares = []
```

Comment:
Would checking `.empty` work instead of the len?

Reply:
Eh, not clearly better as far as I'm concerned